tests/packages/Microsoft.Bot.Components.Telephony.Tests/CallTransferTests.cs

```diff
@@ -21,5 +21,23 @@ public async Task CallTransfer_HappyPath()
         {
             await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer, adapterChannel: Channels.Telephony);
         }
+
+        [Fact]
+        public async Task CallTransfer_EmptyPhoneNumber()
+        {
+            await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer, adapterChannel: Channels.Msteams);
+        }
+
+        [Fact]
+        public async Task CallTransfer_IncorrectPhoneNumber()
+        {
+            await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer, adapterChannel: Channels.Msteams);
```

Comment:
You probably don't want teams channel here

Reply:
yes my bad, copy paste error